src/spikeinterface/extractors/neoextractors/openephys.py

```diff
@@ -180,6 +180,9 @@ def __init__(
             exp_id = exp_ids[0]
         else:
             exp_id = exp_ids[block_index]
+        rec_ids = sorted(
+            list(self.neo_reader.folder_structure[record_node]["experiments"][exp_id]["recordings"].keys())
+        )
```

Comment:
can you check that the keys are str or int ?

Reply:
Yes: https://github.com/NeuralEnsemble/python-neo/blob/master/neo/rawio/openephysbinaryrawio.py#L537